proto/regen/group/v1alpha1/types.proto

```diff
@@ -130,25 +130,28 @@ message GroupAccountInfo {
 message Proposal {
     option (gogoproto.goproto_getters) = false;
 
+    // proposal_id is the unique id of the proposal.
+    uint64 proposal_id = 1;
+
     // group_account is the group account address.
-    string group_account = 1;
+    string group_account = 2;
```

Comment:
groups and group accounts are not the same thing, but we rename it to `group_account_address`,  I agree that would be clearer.

Reply:
Yes, let's rename. We are already in the `Group` module context. Do we need to put `account`? How about `group_address` or `g_address`? It's documented, so the doc string will stay in the structure.

Reply:
we can have several group accounts for the same group so I would find it confusing to just name it `group_address`

Reply:
since we're already in the `GroupAccount` definition, maybe just `address` would be fine?

Reply:
Sounds good. doc comment should be updated as well.

Reply:
We also use `group_account` in `Proposal` and `MsgCreateProposalRequest` so I think it would make sense in those cases to keep having some reference to the group account in the naming, whether we use `group_account_address` or just `account_address`, wdyt?

Reply:
Or maybe we just use `address` everywhere to keep things consistent, since there's only one address that you can provide in these cases, which is the group account address...

Reply:
@robert-zaremba up

Reply:
As long we have a right comment for the struct field it will be good. Let's do that.